hqscripts/management/commands/makemigrations.py

```diff
@@ -51,6 +51,8 @@ def add_arguments(self, parser):
 
     @no_translations
     def handle(self, *app_labels, **options):
+        # used in super().write_migration_files
+        self.written_files = []
```

Comment:
Is this necessary since `super().handle(*app_labels, **options)` is called below?

Reply:
Good catch. I tested running this on 4.2 just to be sure and it isn't necessary. Removed inhttps://github.com/dimagi/commcare-hq/pull/34404